### ISSUE TITLE
Unit test fails

### ISSUE THREAD

**Issue by jvroutak** [eerimoq/bincopy#12]
One test fails on 64-bit Windows 10 host & 32-bit Python:

```
C:\GIT\bincopy>git log -1
commit 2f83b6f1570cb96bb925b416b631112c0ab4283e (HEAD -> master, tag: 14.4.0, origin/master, origin/HEAD)
Author: Erik Moqvist <erik.moqvist@gmail.com>
Date:   Mon Jul 16 08:08:12 2018 +0200

    Version 14.4.0.

C:\GIT\bincopy>git status
On branch master
Your branch is up to date with 'origin/master'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)

        .project
        .pydevproject

nothing added to commit but untracked files present (use "git add" to track)

C:\GIT\bincopy>python2
Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> ^Z


C:\GIT\bincopy>python3
Python 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:06:47) [MSC v.1914 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> ^Z
```

Tests seem to fail:
```
C:\GIT\bincopy>python2 setup.py test
...
======================================================================
FAIL: test_binary (tests.test_bincopy.BinCopyTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\git\bincopy\tests\test_bincopy.py", line 245, in test_binary
    padding=b'\x00'), fin.read())
AssertionError: bytearray(b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00:2001:20010000214601360121470136007EFE09D219012146017E17C20001FF5F16002148011979\r\n:20012000194E79234623965778239EDA3F01B2CA3F0156702B5E712B722B7321460134219F\r\n:00000001FF\r\n') != '\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00:2001:20010000214601360121470136007EFE09D219012146017E17C20001FF5F16002148011979\n:20012000194E79234623965778239EDA3F01B2CA3F0156702B5E712B722B7321460134219F\n:00000001FF\n'

----------------------------------------------------------------------
Ran 61 tests in 2.170s

FAILED (failures=1)
```

```
C:\GIT\bincopy>python3 setup.py test
running test
...
======================================================================
FAIL: test_binary (tests.test_bincopy.BinCopyTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\git\bincopy\tests\test_bincopy.py", line 245, in test_binary
    padding=b'\x00'), fin.read())
AssertionError: bytearray(b'\x00\x00\x00\x00\x00\x00\x00\x[205 chars]r\n') != b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00[188 chars]FF\n'

----------------------------------------------------------------------
Ran 61 tests in 1.912s

FAILED (failures=1)
Test failed: <unittest.runner.TextTestResult run=61 errors=0 failures=1>
error: Test failed: <unittest.runner.TextTestResult run=61 errors=0 failures=1>

```

**Comment by eerimoq** [comment]
Yes, can you add one as you can reproduce the issue?

I have never used a .gitattributes, but I guess `*.bin -text` could work.

https://stackoverflow.com/questions/11162267/how-do-i-make-git-treat-a-file-as-binary

**Comment by eerimoq** [comment]
@jvroutak Any progress, or should I close the issue without a fix?

**Comment by jvroutak** [comment]
I haven't had the time to confirm the fix yet, sorry about that. I should be able to finish this next week though.

**Comment by eerimoq** [comment]
Ok

**Comment by eerimoq** [comment]
Fixed in #13
